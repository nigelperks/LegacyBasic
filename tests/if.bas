1000 A=1
1010 PRINT A
1020 A=A+1
1030 IF A<4 THEN 1010
1100 A$="YOGURT"
1110 IF A$="APPLE" THEN PRINT "FRUIT"
1120 IF A$="YOGURT" THEN PRINT "DAIRY"
1130 IF A$>"WOMBAT" THEN PRINT "HIGH"
