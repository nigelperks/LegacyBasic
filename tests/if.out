 1 
 2 
 3 
DAIRY
HIGH
